COLLABORATOR_GUIDE.md

```diff
@@ -75,47 +78,88 @@ Collaborators or additional evidence that the issue has relevance, the
 issue may be closed. Remember that issues can always be re-opened if
 necessary.
 
+### Author ready pull requests
+
+A pull request that is still awaiting the minimum review time is considered
+`author-ready` as soon as the CI has been started, it has at least one approval,
+and it has no outstanding review comments. Please always make sure to add the
+appropriate `author-ready` label to the PR in that case and remove it again as
+soon as that condition is not met anymore.
+
+### Handling own pull requests
+
+If you as a Collaborator open a pull request, please always start a CI right
+after (see [testing and CI](#testing-and-ci) for further information on how to
+do that) and post the link to it as well. Please also make sure that you start a
+new CI after each update (due to e.g., a change request in a review or due to
+rebasing).
```

Comment:
I'm not 100% sure starting a CI right after opening is the best way to go.  I often wait until there are a few reviews to see if I'm going to have to make any updates.

Reply:
I would like to keep it this way until we get that part automated. Do you feel strongly about it?

See the former discussion for the reasons, e.g., https://github.com/nodejs/node/pull/19116#issuecomment-370453561.

Reply:
@BridgeAR Can we hold off landing this before we have consensus here?

Or, alternatively, could you reword this as a suggestion?

(That is a general feeling I have about this PR, btw: I don’t think we should impose our own workflows on other people.)

Reply:
I personally am not a huge fan about starting a CI always either ^^. The only reason why I think it is good, is that it improves the handling of the PRs as far as I see it.
I also believe that we all want to have a automation of this for collaborators, so I think this is just a intermediate step. I am going to open a issue in build to get some insight in how we might be able to automate it.

This is, by all means, not meant to impose a workflow on someone. Even with the current wording I do not see it as something mandatory, just as something that is really nice to have. I am going to tone down a wording a bit.

Reply:
Reworded.

Reply:
@addaleax @mhdawson is the new wording OK for you? If not: do you have a suggestion how to reword it?